freqtrade/rpc/rpc.py

```diff
@@ -576,6 +589,9 @@ def _rpc_trade_statistics(
             'winning_trades': winning_trades,
             'losing_trades': losing_trades,
             'profit_factor': profit_factor,
+            'winrate': winrate,
+            'expectancy': expectancy,
+            'expectancy_rate': expectancy_rate,
```

Comment:
![image](https://github.com/freqtrade/freqtrade/assets/5024695/18ed088a-12e7-416b-985a-9e534435be49)

i've tested this on my test database (dry run test trades - often with durations of only minutes) - most of them are force-enter / force-exits - and sometimes there's big losers :laughing: 

I struggle to find an explanation for the expectancy values provided.
expectancy-ratio should clearly be negative (this strategy is losing BIG) - but (if i understand this correctly) - it's 4.5 - which i don't really understand.

Reply:
If you use the absolute loss value in the following line does it looks right?

`((loserate / 100) * abs(mean_losing_profit))`

Reply:
OK I've realised my calculations are the wrong way around (expectancy vs ratio). Give me a couple of minutes.